src/ucp/core/ucp_context.c

```diff
@@ -494,10 +507,22 @@ static int ucp_is_resource_enabled(const uct_tl_resource_desc_t *resource,
 
 
     /* Find the enabled UCTs */
-    tl_enabled = ucp_is_resource_in_transports_list(resource->tl_name,
-                                                    (const char**)config->tls.names,
-                                                    config->tls.count, rsc_flags,
-                                                    tl_cfg_mask);
+    if (resource->dev_type == UCT_DEVICE_TYPE_ACC) {
+        /*
+         * Transports for non host memory types are managed by UCX_MEMTYPE_TLS
+         * variable. There are no aliases for memtype transports, check just
+         * real tl names.
+         */
+        tl_enabled = ucp_config_is_tl_enabled((const char**)config->memtype_tls.names,
```

Comment:
why not checking aliases? might as well use the more generic function ucp_is_resource_in_transports_list

Reply:
because we did not define any aliases